adjacent_material.md

```diff
@@ -74,19 +78,14 @@ These parts would require more than one attempt to fully comprehend them
 and some in the group thought it was -- sometimes - difficult to follow
 Jeremy Howard's teaching style.
```

Comment:
The name 'Jeremy Howard' is still mentioned here. Do you want me to remove it?

Reply:
I assumed this was an example of where you were calling out his teaching style specifically. It might be worth clarifying if you were removing/clarifying if you were referring to something more general.